### ISSUE TITLE
zowe.org web page re-design

### ISSUE THREAD

**Issue by Domenico224** [zowe/community#1709]
Here is the draft of the redesign from Phoebe (2 options)
•	Slogan: Combining slogan "Simple, Open, Familiar" in the banner so space got saved and it is more obvious to show on the top.
•	Annoucement: Saving space of the annoucement, which can be expand and collapsed.
•	Intro & journey: Brief intro of Zowe first then followed with the journey of Learn-Use-Extend-Contribute.
•	Zowe components: if we use the layout in option 2, user might be able to see more components in one screen and it could be more friendly for reading. But I understand it costs more development efforts. Option 1 as what it designed now also got its benefits.

Next step show the rendering in a browser

![Option 2.jpg](https://images.zenhubusercontent.com/6290db9e12510e25d3073565/b42a2edb-e7a8-4154-81a7-6eb084934462)![Option 1.jpg](https://images.zenhubusercontent.com/6290db9e12510e25d3073565/921d8f3a-5911-435c-a2c2-8f50c65aa09b)

**Comment by DivergentEuropeans** [comment]
Great job to Phoebe on the work and design! Firstly, I wanted to add I'm not a designer. Take my words with a grain of salt, just my thoughts 🙂 

The things I love: 
- announcements section is visibly slimmer 
- summary comes before the main help sections
- new icons
- prefer version 2 over version 1

Things I think could use some tinkering:
- The tabs don't quite look right to me. Maybe it would look better if the selected tab's selection UI made it a white background with black text? It would make it pop more, what page you are on (rough 45 second sketch below)

![image](https://user-images.githubusercontent.com/20528015/194670195-642e4443-c248-456b-b854-8f92f068c462.png)

- This design has the same issue as our current one and that's we don't respect Introduction to Zowe video. That video is nearly 2 years old and maybe we can make it more up to date w/ all that we've added, but the website should jive better with the video. At the least, the video should take up more right-side space. The left-side text can also be made thinner and taller to probably accommodate a 2x bigger size video w/o compromising the site structure

- And a few nitpicks: the padding between the tiles of "What would you like to do w/ Zowe?" and "Upcoming and recent events" is different, the images for the components have inconsistent shading/shadow schemes

**Comment by Phoebecdy91** [comment]
@DivergentEuropeans Thanks Leanid for the detailed feedback!!
I agree with you that we need to polish the details more for this page. Very good comments and I have marked them down. We can have more further discussion with the whole team and I will iterate the design.

**Comment by DivergentEuropeans** [comment]
@Phoebecdy91 Np. When in doubt, I'd also go towards minimalism. Part of what I like about our website now, it's a got a minimal feel and high contrast colors, it's what it advertises: simple, easy to read. The redesign is pretty good, but it has a "condensed" feel to it too, we need to take care not to clutter it - good luck, reach out to me please on Slack for any more specific opinions if you want!

**Comment by balhar-jakub** [comment]
First I prefer the option 1 as it makes it easier to visualize the projects under Zowe. 

I do like the announcements change, that's making it less prominent. 

I am not even sure the video belongs to the site. I would probably totally remove it and keep it in docs or somewhere else.

**Comment by RASakach** [comment]
Next steps:  Bring proposal to TSC for review / voting for implementation of these zowe.org changes.
Assigned to @RASakach

**Comment by DivergentEuropeans** [comment]
On second and third viewings, agree with @balhar-jakub that option 1 is easier to visualize. Disagree the video should get removed from the site, though I think we should update it instead (understand this is an ANNOYING ask, willing to consider taking this up myself 🙂🙁🙂). I want a video that's no more than 6 min long, I want a brief summary of what Zowe is about, and what the core components bring to the table. More in-depth videos belong in the Docs website

**Comment by nannanli** [comment]
@Domenico-DAlterio @DivergentEuropeans @balhar-jakub @RASakach @Phoebecdy91 
Sam @samanthasusu proposed a new design of the introduction video here: https://github.com/zowe/docs-site/issues/1606#issuecomment-1254760992  Perhaps could be part of this larger revamp effort to record a new shorter version of Zowe overview video focusing on what it is and its value propositions. Then, for intro to each core components, can be a separate series of technical deeper dive videos. 

Deck:
[Zowe overview video redesign draft.pdf](https://github.com/zowe/docs-site/files/9624284/Zowe.overview.video.redesign.draft.pdf)
Intros:
https://user-images.githubusercontent.com/95402892/191709718-1da4bb86-f8dd-4506-8402-c4275a403c52.mp4
https://user-images.githubusercontent.com/95402892/191709701-302008f2-a945-4bab-a167-f0786cfb5e90.mp4

**Comment by RASakach** [comment]
@DivergentEuropeans @Phoebecdy91 @samanthasusu :  Onboarding would like to review the latest proposal for zowe.org changes with you so that we can confirm the latest rev and see if we are all in agreement.  Would it be possible to attend this week's Onboarding Scrum on Friday at 8:00am ET?  Please advise

**Comment by RASakach** [comment]
Hey @Domenico-DAlterio I’m using Zenhub in GitHub, [click this link](https://app.zenhub.com/workspaces/community-5c93e02fa70b456d35b8f0ed/board) to join my workspace and see other features available in GitHub or [download the Zenhub extension](https://www.zenhub.com/extension) and sign up with your GitHub account. 
  _Posted using [Zenhub](https://www.zenhub.com)_

**Comment by Phoebecdy91** [comment]
> @DivergentEuropeans @Phoebecdy91 @samanthasusu : Onboarding would like to review the latest proposal for zowe.org changes with you so that we can confirm the latest rev and see if we are all in agreement. Would it be possible to attend this week's Onboarding Scrum on Friday at 8:00am ET? Please advise

Hi Rosa, thanks for inviting! I will attend this scrum meeting.

**Comment by samanthasusu** [comment]
@RASakach Thank you Rose! I will join this call.

**Comment by DivergentEuropeans** [comment]
Ok, will respond to video draft in the video issue thread: https://github.com/zowe/docs-site/issues/1606#issuecomment-1254760992

**Comment by Phoebecdy91** [comment]
@RASakach here is the updated design. As we discussed in the scrum meeting:
- highlighted tab
- changed previous "Extend" in the tab bar to "App"
- Session of "What would you like to do with Zowe?", refine content of "Create"
- Question for each month: I just make a mockup for using icon and a slight transparent bar for "Question of xxx". It will fixed on the bottom right. It can also be expanded for taking survey. And I will suggest to rephrase the title of the survey, like "One topic a month", "A question a month", keeping it as the tempalte and making user more familiar with what we are doing here.

Comments and feedback are welcome. Let's make it better together. 😄  

<img width="1438" alt="图片" src="https://user-images.githubusercontent.com/23204849/198919715-17c07b15-e093-4f8f-8463-4d88ad312f7f.png">
<img width="1438" alt="图片" src="https://user-images.githubusercontent.com/23204849/198919739-2a9fc98e-055a-4a59-b9d5-0634d259d651.png">

**Comment by DivergentEuropeans** [comment]
Awesome @Phoebecdy91 , looking better! 👍 I'd love to see a few more things:
 - since there's space, main summary content should pad down when announcement is hiding content
 - big, big fan of leaving default (think: first time user) announcement area in single line mode, to be expanded if needed
- the header feels a little tight for me when I view the whole design in full screen mode. I'd like to see the tabs medium wider, maybe tab text a few points bigger, and I'd like to see the header a little/medium taller, to give more breathing room for the tabs and the scrunched logo and text beside the logo. I drew arrows as guides but I'd need to see it to make a good read, use your best judgement. But I think it's too tight the way it is now

I was originally going to send screenshots with arrows, but I spent a bit of time doing that so I just decided to open up Photoshop and make a rough sketch of what I mean! The below image covers the above points and below it I'll add other things I changed:

![design](https://user-images.githubusercontent.com/20528015/199857777-8d25aac3-8577-4c26-8a4e-219f4bfb5d6b.png)

- The roughness of the background in the tabs is unintentional, it's there to save time, but it almost makes me wonder if we can do something about it? Do some experimentation, it doesn't look half bad
- The main first description has been reworded, and the logo text has been added into the description itself
- The spacing in tiles of the bottom section, match the spacing in the tiles of the middle section
- The Question mockup you made has been changed to be square. This follows the tab redesign and the rest of your design so far. Otherwise, this question is the first "round" element on the page and it gives me a subtle but noticeable feeling about it
- The logo has been centered, made larger, and given more space. This is intentionally the ONLY centered element on the page, spatially separating it from our view and telling our brain it's more significant and it is title-like

Let me know your thoughts 🙂

**Comment by DivergentEuropeans** [comment]
Just got off the onboarding call, having a fourth, fifth look at it now, the video is an area I still feel awkwardness/room for improvement - will think about it more in-detail soon... do you have any thoughts?

**Comment by RASakach** [comment]
@Phoebecdy91 we had another great discussion on the latest rev for the zowe.org landing page.  If you have a chance, review the first 30 minutes of the Onboarding meeting recording here: [(https://zoom.us/rec/share/ew4dlNAcrREi9KVWYZ0ymMNMY1-3K10HEa69ftTqg9ViAVKdH6fXD7s0lFV3xxN5.O1M1Jvz693omr9zy)] and let us know if you have any questions.  We agreed with Lenny's suggestions and we had a few more:
(1) change the word ANNOUNCEMENT to ANNOUNCEMENTS 
(2) Change the header UPCOMING AND RECENT EVENTS to **RECENT AND UPCOMING EVENTS** (to match the order in which the events are presented)
(3)  Change the color or the surround box (slightly) for any EVENTS that are in the PAST

@DivergentEuropeans (Lenny), funny how I've been overlooking the placement of that video.  After taking another look, I agree - awkward for sure.  Now that Zowe has some good target market awareness - I feel as though we can move that video behind the LEARN option and place it front-and-center on the page that appears when the website visitor clicks on LEARN.  What do you think?  If you agree, can you try playing around with that page?

**Comment by DivergentEuropeans** [comment]
@RASakach I'm alright with keeping the video on the front-page for now in combo with if we can think of a better way to present it - one idea: increase video size, decrease left-side text width by a little, and remove video border for a larger video preview & player. I'd rather have it on the front-page than not, and my opinion will mature depending on the quality of our new video. Final result? Something like this, see if you like it @Phoebecdy91 

![design2](https://user-images.githubusercontent.com/20528015/200608746-191eabb7-c87f-463f-a52a-3c6a62b67739.png)

I presented internally to App Framework squad what our current version of the front-page looks like just to get some fresh eyes into the room, that got us thinking about priorities on what to show on the front itself and @1000TurquoisePogs had some great points that I loved, I want to see them critiqued or integrated into our redesign:

- A very common user question, one of the most common, is "When is the next release?". We have info about what the current release is, or new release, but not when the next one will be and we think it should be on the front page

- The zowe-dev calendar is more important to the average user (our opinion: willing to discuss) than the Recent and Upcoming Events. If a user has a question or suggestion about Zowe, and if they see from the front page "Oh, there's this and so-and-so meeting on Friday, let me go to it!" this is great value and could help boost our attendance for every squad. Recent and Upcoming Events can stay, just moved down somewhere

**Comment by Phoebecdy91** [comment]
Hi @RASakach @DivergentEuropeans I heard the recording last night! Great discussion! Several updates and feedback here:
<img width="1710" alt="image" src="https://user-images.githubusercontent.com/23204849/200892295-4052c424-0bb0-4eb9-996d-98476d42ba6a.png">

**Tabs:**
- A pane background for the tabs is really cute. I think we can try that to separate banneer and tabs. That would be a good idea. I also changed the selected tab background as blue, so it will be less monotonous that we overall have white and black on the top.
- I tried to zoom in to full screen for both design version. If we put the tabs bar to fit the width of the webpage, it might case a problem when the user's screen is too wide. It will make each tab too far away. Thus I would suggest to keep it in the center, giving the maximun width as e.g.1024px. So as responsive, I think we can set the rules like, if the screen width is smaller than 1024px, tab would be fill into the width. If screen is wider than 1024 px, keep tab bar as 1024 px.
- I do like the simper version of slogan + logo that makes our branding more attractive. But it seems that Lenny's reversion will take higher space of the webpage. I might still need to work on that.

**Introduction of Zowe:**
- key content got updated as Lenny suggested. But feel free to let me know if we got new version.
- New idea: If we would love to make the video area bigger and match with the content size, maybe have greetings as the title of this section. (I am not sure whether emoji 👋 is fine to this webpage or not. Will it be too less serious？ 😂)
- New idea: ending sentence to show connection with the following content, like : start your journey...

**Annoucements:**
- folded by default in single line mode and expand it when need makes sense to me. 
- I didn't get changes for this one. But if any other thoughts, let me know.

<img width="1695" alt="image" src="https://user-images.githubusercontent.com/23204849/200892858-a2580215-9101-48b0-83ed-d588109d38ec.png">
**Recent and upcoming events:**
- I think gray-out the tile might be a little bit dangerous since this tile can still be click to the event page.
- Wondering whether it would be good that we have a small bell as a icon in front of the event date. Kind of like a reminder or attracting people to pay attention to this date. Make it as a place holder here to see if we have better idea.

<img width="1715" alt="image" src="https://user-images.githubusercontent.com/23204849/200893673-a1786008-27e9-40a4-bea9-216a3677db49.png">
**Survey:**
- The reason I am giving it as a oral style and in the latest design I redesign this part more like a badge. I hope it could be a little bit different from what we have in the whole Zowe page. Looking for differences might bring things new to visitors.
- And I see Lenny's point about the square will be more consistent with the whole Zowe page style.
- Both two opinions have their advantages. So maybe we can discuss within squad to see how people feel about the changes.

Plus, I am so sorry that I were not able to join our last call and our studio is gonna to have team building this Friday + weekend, I will be out of office. I cannot join our call again! I feel so sorry about it.
You guys can discuss it first. Or I can join next week to have detailed discussion or quick iterate with you. THX!

**Comment by balhar-jakub** [comment]
@DivergentEuropeans I fully agree with when the next release will be. We need to answer this on the front page. 

As for the zowe-dev calendar, I don't think we should directly embed it on the front page. If we want it to be useful we should figure out a good way to show the key events coming out of it.

**Comment by DivergentEuropeans** [comment]
Yes @balhar-jakub something like a "this week's meetings" or some smarts to show the next meetings for the next 4-5 business days

**Comment by RASakach** [comment]
@Phoebecdy91 Thanks again, for your support in this redesign!  Here are the results / requests from today's Onboarding Meeting:

1. When selected, the TABs should be WHITE, not blue (landing page tab = OVERVIEW) as shown in Lenny's mock-up above
2.  The LOGO at the very top looks great however the words OPEN SIMPLE FAMILIAR should have a period "." after each word as shown in Lenny's mock-up above
3.  Remove "Hi, Welcome to Zowe"; we feel this looks less professional, your hunch was correct.
4.  The Video placement is good, however, the border should be white, not black (see Lenny's mock-up)
5.  in the "Recent and upcoming events" section, it was a great thought to add the bell icon, however we feel that the visitor may associate the bell with the ability to register; can we try to change the color of the titles from blue to gray for past events?
6.  we like your rendition of the question-of-the-month "banner", let's keep that
7.  Although we like the "excited" nature of it, we agreed to eliminate the line "For the OPEN, SIMPLE, FAMILIAR experience, let's get started!"  The statement takes up more real-estate on the page and we feel the next header "What would you like to do with Zowe" achieves the same goal.

We discussed additional changes but we don't want to move on those yet because we want to get the next rev out as soon as possible.  

Here is the link to the recording if you are interested:  https://zoom.us/rec/share/R579zezkPCnWv-46Wgtxjnx_zFW319K4L0EIVw5EZc0Hfks_1_PV1-u-urDKSeHb.rvTR6fJB3NTYXZKL

@samanthasusu:  I apologize - I did not realize you were joining us today.  We briefly reviewed the video at the very end of our meeting and a few of us agreed to review it again over this coming week and send you recommendations.  Thank you for all of your work on this!

**Comment by samanthasusu** [comment]
> @Phoebecdy91 Thanks again, for your support in this redesign! Here are the results / requests from today's Onboarding Meeting:
> 
>     1. When selected, the TABs should be WHITE, not blue (landing page tab = OVERVIEW) as shown in Lenny's mock-up above
> 
>     2. The LOGO at the very top looks great however the words OPEN SIMPLE FAMILIAR should have a period "." after each word as shown in Lenny's mock-up above
> 
>     3. Remove "Hi, Welcome to Zowe"; we feel this looks less professional, your hunch was correct.
> 
>     4. The Video placement is good, however, the border should be white, not black (see Lenny's mock-up)
> 
>     5. in the "Recent and upcoming events" section, it was a great thought to add the bell icon, however we feel that the visitor may associate the bell with the ability to register; can we try to change the color of the titles from blue to gray for past events?
> 
>     6. we like your rendition of the question-of-the-month "banner", let's keep that
> 
>     7. Although we like the "excited" nature of it, we agreed to eliminate the line "For the OPEN, SIMPLE, FAMILIAR experience, let's get started!"  The statement takes up more real-estate on the page and we feel the next header "What would you like to do with Zowe" achieves the same goal.
> 
> 
> We discussed additional changes but we don't want to move on those yet because we want to get the next rev out as soon as possible.
> 
> Here is the link to the recording if you are interested: https://zoom.us/rec/share/R579zezkPCnWv-46Wgtxjnx_zFW319K4L0EIVw5EZc0Hfks_1_PV1-u-urDKSeHb.rvTR6fJB3NTYXZKL
> 
> @samanthasusu: I apologize - I did not realize you were joining us today. We briefly reviewed the video at the very end of our meeting and a few of us agreed to review it again over this coming week and send you recommendations. Thank you for all of your work on this!

@RASakach Hi Rose, thanks for having this discussion and you don't have to apologize, I was intended to bring this up after the homepage redesign discussion. But I was called to do the covid test so I dropped. 
I will review all your comments this week and upload the completed demo before this Fri. You could also leave your comments in the [storyboard](https://ibm.ent.box.com/notes/1057188686530) to modify the script. And I will try to make the discussion this Friday.

**Comment by Phoebecdy91** [comment]
@RASakach Thanks for the feedback and comments.
- design updated for the tabs, slogan, intro content
- for the black boarder of the video, it is not designed by me😂 it was auto styled when the screen narrow to certain width. I guess this is the thumbnail style from Youtube.
- For the events, graying out the title will make this tile look unclickable. I know user can go to the event web through clicking the title. I made some changes: 
> - gray check icon + gray content of the date which means this event has passed. 
> - Green icon + black content of date means is comming soon or ongoing.

We can have discussion later today.
<img width="1062" alt="image" src="https://user-images.githubusercontent.com/23204849/202598125-e4416490-0912-48ff-a82c-92ca863350d2.png">

**Comment by RASakach** [comment]
@Phoebecdy91:  Thank you for reviewing the latest zowe.org landing page mock-up with us today.

@DivergentEuropeans (Lenny):  FYI we are CLOSE to being ready to implement this round of updates, however we'll need "developers" to assist @nannanli  (Ashley) and the Doc squad with the implementation. (Phoebe reminded us that the black boarder around the video is something she can't change but I recall Lenny indicating it can be removed programmatically). Assuming Ashley will be prioritizing the webpage development work at some point?  How do we want to manage that?

Here's what's left to do:
(1) Finalize the new rev of the VIDEO*  [this should not hold up implementing our latest changes]
(2) Revise the descriptions for EACH of the Zowe projects (in the tiles) such that each tile has roughly the same amount of verbiage
(3)  Determine where to place the LINK to the current Zowe Roadmap

My proposal for #3 would be to place a sentence at the bottom of this area.  (beneath the last paragraph)

"CLICK here for the latest Zowe roadmap"

![ZoweRoadmapLinkProposal.PNG](https://images.zenhubusercontent.com/5cf586b9184a60764b0fb167/52e6d3bf-15b3-4f06-9481-1aef4d73932d)

**Comment by DivergentEuropeans** [comment]
Ooh, here's another thing! Our vNext page doesn't currently mean "vNext". This tab needs to be renamed to "V1 to V2" right?

**Comment by RASakach** [comment]
@DivergentEuropeans, @balhar-jakub @zFernand0 @JillieBeanSim  FYI,  I have created a document with some verbiage associated with Zowe CLI and Zowe Explorer.  Please review when possible.

The first section is an attempt to reword the description listed in the TILE for the component on the zowe.org landing page.
The second section is an attempt to call out the most important points that should be mentioned in the Zowe INTRO VIDEO.

Here's the LINK:  https://ibm.box.com/s/f02rdoafky0b7hqlvxqtpr3435900mv7
(change tracking is on).

**Comment by RASakach** [comment]
@DivergentEuropeans :  We are ready to move forward with this "rev" of the Zowe.org website redesign.  We like your suggested mock-up that includes the "next release" and the link to the roadmap, with 2 minor adjustments:
(1) Replace "Next release" with "Next _planned_ release"
(2) Change the format of the date for the "Next planned release" to yyyyMMMdd (example:  2022DEC12) to eliminate any chance of confusion over which number is the MM  and which number is the DD

[For the roadmap, we will be posting it to the website Github repo and will make that link available to you when we have it]

Assuming you can take the lead on implementing these changes?  (We realize it is pending your capacity - any idea on an estimated ETA? We do not want to wait for the revised intro video.) TY so much for all of your help with this!
FYI:  @samanthasusu @Phoebecdy91 @nannanli

**Comment by DivergentEuropeans** [comment]
Ok, added my App Framework points to the box note. Formatting appears a little wonky with the MSWord Online, but content is there.

Like both your points @RASakach sounds great to me
> Assuming you can take the lead on implementing these changes?
I don't mind taking the lead, but I'll say that the time I have to contribute to this will be very spaced out, as we have a roadmap we've committed to already internally. That being said, definitely can work on this slowly over the coming months. And these changes aren't ground-shaking so it shouldn't be too bad. So first I'll ask, are there any web developers who can help me with this in the onboarding squad?

> estimated ETA
There's a few things I still don't see in this final version, assuming my latest comment is more or less the final thing: 
1) we don't have the meetings calendar embedded 
2) we don't see the rest of the pages, just the homepage. 
If I get time, I'll make a mockup of meetings calendar, but we can move on if needed from the design stage at this point, as that stuff can fall together when development begins - I'm cool with this. I'll give an ETA when I know if I have any help or not. (Very wild guess, as I have worked very little with the Zowe website repo) Assuming no help, and it's just me, say I put in two hours a week, this is probably a 20 hour job including redesigning all pages to fit the new theme, so 10 weeks seems reasonable

**Comment by balhar-jakub** [comment]
I guess, I am the only other person within the Onboarding squad that is at least somewhat developer. I can try to help but my capacity is very limited.

**Comment by RASakach** [comment]
Onboarding agreed today that @DivergentEuropeans and @balhar-jakub will take the lead on implementing this re-design work.

ETA:  end-of-February

**Comment by balhar-jakub** [comment]
@DivergentEuropeans @Phoebecdy91 Can any of you help me by providing the background image for the top part of the web page and the images for Question of the month and for the Learn, Use, Create, Contribute?

**Comment by balhar-jakub** [comment]
The current state of work is visible on this URL: https://deploy-preview-704--condescending-dubinsky-4645a9.netlify.app/ 

The URL is from PR.

**Comment by Phoebecdy91** [comment]
@RASakach @balhar-jakub @DivergentEuropeans Sorry for replying late! It is so glad that we have team to implement this.
For the background image, I am using the one in our current webpage. So I don't have the orignal one.
And here is the link for Figma file, I think it could be better help you to see some inspect propotities of each component, like the CSS, size, color... And you can also export some image, icons, through this file as well.
https://www.figma.com/file/nloQ7TDh59YAKKWCXIKw0C/Zowe.org?node-id=301%3A3012&t=FfRPko5fDSNnkN16-1
I have invited Jakub and Rose to this file but cannot find Lenny's Email address. Feel free to let me know so I can add you to this file. And if this link doesn't work for you, please tell me.

**Comment by RASakach** [comment]
We are completing the final steps of our phase 1 update
ETA Feb 2023

**Comment by Phoebecdy91** [comment]
Thanks @balhar-jakub I saw your [implementation](https://deploy-preview-704--condescending-dubinsky-4645a9.netlify.app/)!! It looks really great! 

Just some slight adjustment for the spacing and font size. I am sorry that making this work repeated, but hopefully it can be helpful to make it looks better.

- I would suggest that we change each section title font to **1.75 rem**, I think now is 2rem, it looks more suitable for the whole page.

The tile for Learn, Use, Create, Contribute, might need some changes:
- if the padding could be changed to 20px, that would looks better:
```
 #intents div.row > div {
  padding: 20px;
  margin: 10px;
}
```

- font size and format:
```
#intents div.row > div a {
  font-weight: inherit;
  font-size: xx-large;
  padding-left: 15px;
  vertical-align: bottom;
}
```
- image size: 48 px
<img width="1509" alt="图片" src="https://user-images.githubusercontent.com/23204849/216263182-1dbb6769-4ac9-4637-bf68-50bed0f94175.png">


As the tile for each component:
- The font size of the title could be 1.25rem
- the padding of the tag could be 4px, and I am wondering whether it is ok to make differences for left & right padding as maybe 8px, and up & bottom padding for 4px.
```
span.long-term-support {
  background-color: #0059FF;
  border-radius: 22px;
  color: white;
  padding: 4px;
}
```
- for the picture of each component, can we set it as same size, even triming the the picture is fine for me. So it can look more tidy for different tiles in one screen.
<img width="922" alt="图片" src="https://user-images.githubusercontent.com/23204849/216319195-c0452246-0dbb-40a2-ac06-0a92c94038ea.png">

And one more quick question, is that possible for us to make it responsive for this page? For example, if the browser is wider, we can show 4 tiles in one row. If narrow, maybe 2 tiles in a row.

So sorry that I left too many comments here. Feel free to contact me if we need some prioritization. I can work with you for that.

**Comment by balhar-jakub** [comment]
@Phoebecdy91 I will try to address them within the coming two weeks.

**Comment by balhar-jakub** [comment]
@Phoebecdy91 All the changes are reflected apart from more responsiveness. The current state is that the projects are responsive in a limited fashion. When the screen is below 750px there is just one column of them.

Making them more responsive would require significant effort. I believe it would be good to have this as a separate issue so it doesn't delay the update to the zowe.org even further.

**Comment by balhar-jakub** [comment]
The latest PR contains all the changes proposed in this original proposal. As such I am looking for review and then we can merge the changes.

**Comment by Phoebecdy91** [comment]
Thanks @balhar-jakub for the implementation! Looks great! And I agree that we can separate it as another issue for the responsive design. And feel free to let me know if anyone who will implement the responsive design, I can provide more details on the tiles for when to be 1 or 2 or 3 or even 4 column regarding to specific width of the page.

And one more quick comment, sorry that I didn't notice that last time.
When the announcement section got expanded, the arrow should point up, which means it can be folded.
<img width="413" alt="图片" src="https://user-images.githubusercontent.com/23204849/218769185-19158942-4c08-4060-911a-8fa7bb5d75b2.png">
So when the announcement is folded, the arrow should point down like a dropdown arrow.
<img width="279" alt="图片" src="https://user-images.githubusercontent.com/23204849/218769448-178dc2b3-c94f-4cdf-921a-b908b7bc66df.png">

Thanks a lot!

**Comment by balhar-jakub** [comment]
@Phoebecdy91 Please create new issue for responsiveness and assign it to me. I will handle this once the current list of changes is merged. 

The change for the icon is now in the PR.

**Comment by Phoebecdy91** [comment]
@balhar-jakub Thanks Jakub! I might need some more time to prepare detailed design files for responsive design. I will create the issue once it is ready.

**Comment by RASakach** [comment]
TSC group reviewed / agreed to publish this iteration of the Website redesign to make it LIVE for the SHARE conference next week.
